src/reducers/proxy.js

```diff
@@ -69,15 +73,18 @@ const handleTx = ({
   acctType
 }) =>
   new Promise(resolve => {
+    console.log('handleTx');
     const txMgr = window.maker.service('transactionManager');
     txMgr.listen(txObject, {
       pending: tx => {
+        console.log('handleTx pending');
         dispatch({
           type: `proxy/${prefix}_SENT`,
           payload: { txHash: tx.hash }
         });
       },
       mined: async _ => {
+        console.log('handleTx mined');
```

Comment:
I need to look more into why the loading spinner isn't popping up when the tx begins, and doesn't resolve when its been mined.